velox/functions/sparksql/Arithmetic.h

```diff
@@ -350,4 +350,63 @@ struct ToHexBigintFunction {
     ToHexUtil::toHex(input, result);
   }
 };
+
+FOLLY_ALWAYS_INLINE static int8_t fromHex(char c) {
```

Comment:
Let's put this function into 'detail' namespace.

Reply:
ok, thanks!